dfs/nfs/mounts_test.go

```diff
@@ -11,7 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build unit
+// +build unit, linux
```

Comment:
My general concern is the further apart the platforms vary in their testing, the less valuable the non-linux platforms become. Maybe I'm confused about the purpose of this PR and why we want to be able to (not run but) test on OSX. Certainly you can install NFS on OSX. (Not sure about devmapper.)